ftw/mobile/templates/buttons_viewlet.pt

```diff
@@ -1,6 +1,7 @@
 <div id="ftw-mobile-wrapper">
     <nav id="ftw-mobile-menu-buttons"
          tal:attributes="data-navrooturl string:${view/nav_root_url};
+                         data-portaltitle string: ${view/portal_state/portal_title};
```

Comment:
Is this compatible with custom navigation roots / subsites?

Reply:
It is. It's the fallback if the node is not composed out of a brain. Which is only the case for the plone site root. subsites are in the catalog, thus have a title.